### ISSUE TITLE
Specifying test(s) to run at verdi devel tests

### ISSUE THREAD

**Issue by szoupanos** [aiidateam/aiida-core#2772]
This is a feature that would make the debugging much easier.
It would be nice to be able to specify when running verdi devel tests a set of tests that we would like to run (from the specified "family" of tests)

E.g.

verdi -p test_dj1 devel tests db.nodes

has 71 tests. I want to focus only on one and debug it but I don't want to run all the other since it takes time (to run them on every modification). It becomes even more cumbersome if the whole procedure is done via the debugger of an IDE with breakpoints etc.

This feature exists  for unittests, we just have to "expose it'
https://stackoverflow.com/questions/15971735/running-single-test-from-unittest-testcase-via-command-line

**Comment by ltalirz** [comment]
This woudl be solved by #2771

**Comment by ltalirz** [comment]
And now it is solved - use e.g.

```
# tests from a module
pytest tests/path/to/test_mystuff.py

# single test
pytest tests/path/to/test_mystuff::test_func
```
And [there's more](https://docs.pytest.org/en/latest/usage.html#specifying-tests-selecting-tests)
